cmd/fleetctl/debug.go

```diff
@@ -338,18 +349,23 @@ func debugArchiveCommand() *cli.Command {
 				}
 				fmt.Fprintf(os.Stderr, "Ran %s\n", profile)
 
+				outname := profile
+				if ext != "" {
+					outname = profile + "." + ext
+				}
+
 				if err := tarwriter.WriteHeader(
 					&tar.Header{
-						Name: outfile + "/" + profile,
+						Name: outfile + "/" + outname,
 						Size: int64(len(res)),
 						Mode: defaultFileMode,
 					},
 				); err != nil {
-					return fmt.Errorf("write %s header: %w", profile, err)
+					return fmt.Errorf("write %s header: %w", outname, err)
```

Comment:
Shouldn't the error here now contain: `outfile + "/" + outname`? (Same below.)

Reply:
yeah, I think your suggestion will make the error way more clear. Pushed a fix.